tests/shell/tests/01-run.py

```diff
@@ -79,6 +86,20 @@ def check_and_get_bufsize(child):
     return bufsize
 
 
+def check_line_exceeded(child, bufsize):
+
+    if BOARD == 'nrf52dk':
+        # looks like the nrf52dk runs in to undefined behaviour when sending more
+        # than 64 bytes over UART
+        print_error('test case "check_line_exceeded" broken for nrf52dk. SKIP')
+        return
```

Comment:
Maybe it is only a local issue on my machine, so it would be interesting whether it works for other people. On my machine, the nrf52dk is not able to process more than 64 bytes correctly over UART (when using the stdio_nimble module everything works fine though). Meanwhile everything works fine with a nucleo-f334r8.

Reply:
I'm also getting a weird behaviour here:

```
> UsssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssUUUUU
USBS�ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssUSBS�
```

`USB` appear for now reason??

Reply:
Someone should connect a known-good USB-to-serial converter to actual nrf52dk uart outputs. I suspect the onboard debugger is making trouble here.

Reply:
Good catch, no issues when connecting a serial uart and with the `viartual com absent` (USB not connected):

```
> end_test
end_test
[TEST_END]
> 
make: Leaving directory '/home/francisco/workspace/RIOT/tests/shell'
```

```diff
diff --git a/tests/shell/tests/01-run.py b/tests/shell/tests/01-run.py
index bbe93d78db..ec5e9bbbd8 100755
--- a/tests/shell/tests/01-run.py
+++ b/tests/shell/tests/01-run.py
@@ -76,6 +76,7 @@ def check_cmd(child, cmd, expected):
 
 
 def check_startup(child):
+    child.sendline('reboot')
     child.expect('test_shell.\r\n')
 
 
@@ -88,15 +89,7 @@ def check_and_get_bufsize(child):
 
 
 def check_line_exceeded(child, bufsize):
-
-    if BOARD == 'nrf52dk':
-        # looks like the nrf52dk runs in to undefined behaviour when sending more
-        # than 64 bytes over UART
-        print_error('test case "check_line_exceeded" broken for nrf52dk. SKIP')
-        return
-
     longline = "_"*bufsize + "verylong"
-
     child.sendline(longline)
     child.expect('shell: maximum line length exceeded')
 

```

Reply:
Interesting, I will remove the error message then. But what about a warning message? There may be other boards with similar problems, so a warning could save quite some time for a developer who runs in to the same issue.

Reply:
The problem is nrf52dk is used by the CI as well, so the test should be skipped. It would be better to know why the debugger causes this though... I looked around a little this morning but no idea.... Maybe @haukepetersen has an idea since he works a lot with this board?.

An issue could be a good idea as well, not sure how this relates to #10639

So I think we need to keep an error message... I'll run the test on multiple boards to see if other show problems, I had no issues with `nrf52840` for example.

Reply:
tested on multiple boards and there is no issue